stp_core/config.py

```diff
@@ -23,12 +23,15 @@
 # Enables/disables debug mode for Looper class
 LOOPER_DEBUG = False
 
-
-# Zeromq configuration
+# Quotas configuration
+ENABLE_DYNAMIC_QUOTAS = False
+MAX_REQUEST_QUEUE_SIZE = 10000
```

Comment:
Isn't it too much? If we have a load of 10 writes per sec, than the client stack will be stopped if we don't order anything for 1000 secs.